rosbridge_server/scripts/rosbridge_websocket_client.py

```diff
@@ -199,12 +226,21 @@ def end_video(self):
 
         # Connect with server
         server_uri = rospy.get_param("~webserver_uri")
-        enable_authentication = rospy.get_param(
-            '~enable_authentication', False)
         # In the future we are going need to use everithing on the same port
         # given throught the argument
         ws = WebsocketClientTornado(server_uri)
 
+        # Authentication options
+        authentication_method = rospy.get_param('~authentication_method', None)
+        if authentication_method.find(AUTHENTICATION_ARG_MAC):
+            authenticate_mac = True
+            rospy.loginfo("Authentication method using MAC address")
```

Comment:
It's not {MAC address](http://en.wikipedia.org/wiki/MAC_address), it's simply [MAC](http://en.wikipedia.org/wiki/Message_authentication_code) (message authentication code)